lang/sr.json

```diff
@@ -8,7 +8,7 @@
 	},
 	"dir": "ltr",
 	"LangCode": "sr",
-	"DateTime": "d.m.yyyy HH:MM",
+	"DateTime": "m/d/yyyy h:MM aa",
```

Comment:
You're sure about this one? [Wikipedia claims](https://en.wikipedia.org/wiki/Date_and_time_notation_in_Serbia) that Serbian date & time format is `d.m.yyyy HH:MM` actually? I'm also not sure what "aa" would mean...